File: Repositories/CSI-Java-2021-01/CSI-Pedro-Irene/MainProject/src/csi/irene/pkg/RockPaperScissors.java
package csi.irene.pkg;
import java.util.*;
public class RockPaperScissors {
	public static void main(String[] args) {
	
	
	String[] choices= {"Rock", "Paper", "Scissors"};
	Random r=new Random();
	//int randomInt = r.nextInt(choices.length)
	String computerChoice = choices[r.nextInt(choices.length)];
	
	Scanner sc = new Scanner(System.in);
	
	//int inputInt = sc.nextInt();
	System.out.print("Type an index: ");
	String userChoice = choices[sc.nextInt()];
	
	System.out.println("Computer chose: " + computerChoice);
	System.out.println("User chose: " + userChoice);
	
	if(computerChoice == userChoice) {
		System.out.println("Its a Tie!");
	}
	else if(computerChoice == choices[0] && userChoice == choices[1]) {
//		Computer chose Rock and user chose Paper
		System.out.println("You Win!");


	}
	
	else {
		System.out.println("Something went wrong!");
	}
	
	 if (computerChoice == choices[0] && userChoice == choices[2]) {
	System.out.println("You Lose!");
	
	}
	
	 else if(computerChoice == choices[1] && userChoice == choices[0]) {
	System.out.println("You lose!");
	 }
	if(computerChoice == choices[1] && userChoice == choices[2]) {
	System.out.println("You Win!");
	}
	else if(computerChoice == choices[2] && userChoice == choices[0]) {
		System.out.println("You retar!");
	}
	if(computerChoice == choices[2] && userChoice == choices[1]) {
	System.out.println("Very Good");	
	}
	}
}

File: Repositories/CSI-Python-2021-02/CSI-Francisco-Ruiz/Modules/Module5/RockPaperScissors.py
import random

foo = ["Rock", "Paper", "Scissors"]
computerChoice = random.choice(foo)
userChoice = input("select Rock, Paper or Scissors ")
print(computerChoice)

if(userChoice == "Rock" and computerChoice == "Paper"):
    print("You lose")
elif(userChoice == "Rock" and computerChoice == "Scissors"):
    print("You win")
elif(userChoice == "Rock" and computerChoice == "Rock"):
    print("Tie")
elif(userChoice == "Paper" and computerChoice == "Scissors"):
    print("You lose")
elif(userChoice == "Paper" and computerChoice == "Paper"):
    print("Tie")
elif(userChoice == "Paper" and computerChoice == "Rock"):
    print("You win")
elif(userChoice == "Scissors" and computerChoice == "Rock"):
    print("You lose")
elif(userChoice == "Scissors" and computerChoice == "Scissors"):
    print("Tie")
elif(userChoice == "Scissors" and computerChoice == "Paper"):
    print("You win")
else:
    print("Rock beats Scissors, Scissors beats Paper, and Paper beats Rock")
